.github/scripts/visualize_stopping_patterns.py

```
@@ -0,0 +1,439 @@
+    for section in group_sections:
+        section_length = len("\n") + len(section)
+        if current_length + section_length + len(truncation_msg) > MAX_COMMENT_LENGTH:
+            parts.append(truncation_msg)
+            break
+        parts.append(section)
+        current_length += section_length
+
+    comment = "\n".join(parts)
+
+    # Write output
+    output_path = "/tmp/visualization_comment.md"
```

Ruff (0.15.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/visualization_comment.md"

(S108)